src/Miningcore/Payments/PayoutManager.cs

```diff
@@ -121,18 +114,37 @@ public void Start()
                             logger.Error(ex, () => $"[{pool.Id}] Payment processing failed");
                         }
                     }
-                    //}
-
-                    //catch(Exception ex)
-                    //{
-                    //    logger.Error(ex);
-                    //}
 
                     await Task.Delay(interval, cts.Token);
                 }
             });
         }
 
+        public async Task<string> PayoutSingleBalanceAsync(PoolConfig pool, string miner)
+        {
+            var success = true;
+            var timer = System.Diagnostics.Stopwatch.StartNew();
+            Decimal amount = 0;
+            IPayoutHandler handler = null;
+            try
+            {
+                handler = await ResolveAndConfigurePayoutHandlerAsync(pool);
+                var balance = await cf.Run(con => balanceRepo.GetMinerBalanceAsync(con, pool.Id, miner));
+                amount = balance.Amount;
+                return await handler.PayoutSingleBalanceAsync(balance);
+            }
+            catch (Exception ex)
+            {
+                logger.Error($"Failed to payout {miner}: {ex}");
+                success = false;
+                throw;
+            }
+            finally
+            {
+                TelemetryUtil.GetTelemetryClient()?.GetMetric("FORCED_PAYOUT", "success", "duration").TrackValue((double)amount, $"{success}", $"{timer.ElapsedMilliseconds}");
```

Comment:
You can just put success or success.ToString() if it expects string type. Using string interpolation($) everywhere is unnecessary.